src/Middleware/ResponseCompression/src/ResponseCompressionBody.cs

```diff
@@ -15,35 +16,48 @@ namespace Microsoft.AspNetCore.ResponseCompression
     /// <summary>
     /// Stream wrapper that create specific compression stream only if necessary.
     /// </summary>
-    internal class BodyWrapperStream : Stream, IHttpBufferingFeature, IHttpSendFileFeature, IHttpResponseStartFeature, IHttpsCompressionFeature
+    internal class ResponseCompressionBody : Stream, IHttpResponseBodyFeature, IHttpsCompressionFeature
```

Comment:
Why doesn't this use SendFileFallback?

Reply:
You're right, that's more existing code I can delete.